docs/media-buy/api-reference.md

```diff
@@ -712,12 +714,47 @@ Lists available advertising products for the authenticated principal with option
       "is_custom": false,
       "brief_relevance": "Premium CTV inventory aligns with sports content request and prime time targeting"  // If brief was provided
     }
-  ]
+  ],
+  "policy_compliance": {
+    "status": "approved",
+    "message": "Advertiser and products approved for all inventory"
+  }
 }
 ```
 
 **Note**: If no brief is provided, returns all available products for the principal.
 
+**Policy Compliance Response:**
+When products array is empty due to policy restrictions, the response includes:
+
+For advertisers that cannot be supported:
+```json
+{
+  "products": [],
+  "policy_compliance": {
+    "status": "blocked",
+    "message": "Alcoholic beverage advertising requires age-gated inventory. This publisher does not support age verification."
```

Comment:
I think the advertiser should be communicating what their requirements are here, right? should we add a field where the advertiser specifies any compliance requirements, and the pub would say "compliance": "not met"? or something?